plotter/heat.go

```diff
@@ -98,6 +98,11 @@ func (h *HeatMap) Plot(c draw.Canvas, plt *plot.Plot) {
 	}
 	// ps scales the palette uniformly across the data range.
 	ps := float64(len(pal)-1) / (h.Max - h.Min)
+	// for some degenerate data sets with h.Max == h.Min, this blows
+	// up and needs to be caught
+	if math.IsNaN(ps) {
```

Comment:
`h.Min` and `h.Max` can be set prior to plotting. If a selection of heatmaps are being rendered, they should be on the same scale and so should be being set to have the same range.